Yoast/yoast-components
# Add conditional text field

This PR adds required/conditional text fields to the wizard. These fields are an extension on the normal TextField and they only show when another field has a required value.

Testing:
1. Start the wizard.
2. Go to step 4 `Company or Person`
3. See if nothing is rendered initially.
4. Change the choice input to person and see if the TextField for a person appears.
5. Change the choice input to company and see if the TextField for company is rendered and the person TextField isn't.
6. Put in some values and see if they are remembered when you change the choice and/or the step.
7. Run the tests.

Fixes #33
